resmon

```diff
@@ -77,7 +79,15 @@ unless($debug) {
     open(STDIN, "</dev/null");
     open(STDOUT, ">/dev/null");
     open(STDERR, ">/dev/null");
-    fork && exit;
+    my $pid=fork;
+    if ($pid) {
+      if (my $PIDFILE=$config->{pidfile}) {
+        open(PIDFILE,">$PIDFILE") or die "can't open >$PIDFILE";
+        print PIDFILE $pid;
+        close PIDFILE or die "can't close $PIDFILE";
+      }
+      exit;
+    }
```

Comment:
Please use the three-arg form of `open()`, instead of the two-arg form with string interpolation, and scalar filehandles instead of barewords:

`open(my $pidfh, '>', $config->{pidfile}) or die "can't open $config->{pidfile}";`